python/pyxet/pyxet/url_parsing.py

```diff
@@ -181,7 +181,7 @@ def parse_url(url, default_domain=None, expect_branch = None, expect_repo = True
         ret.domain_explicit = False
         explicit_user = None
     else:
-        domain_user = netloc.split(":")
+        domain_user = netloc.rsplit(":", maxsplit = 2)
```

Comment:
```suggestion
        domain_user = netloc.rsplit(":", maxsplit = 1)
```

Reply:
https://www.w3schools.com/python/ref_string_rsplit.asp#:~:text=%23%20setting%20the%20maxsplit%20parameter%20to%201%2C%20will%20return%20a%20list%20with%202%20elements!

Reply:
Oh, I see. Of course.

Reply:
Fixed.